src/Blocks/components/form/assets/index.js

```diff
@@ -4,123 +4,153 @@ import domReady from '@wordpress/dom-ready';
 import { componentJsClass } from './../manifest.json';
 import { FORM_EVENTS, FORM_SELECTORS, FORM_DATA_ATTRIBUTES } from './form';
 
+if (typeof esFormsLocalization === 'undefined') {
+	throw 'Your project is missing global variable esFormsLocalization called from the enqueue script in the forms.';
+}
+
 const selector = `.${componentJsClass}`;
 
+// You can disable auto init from the admin.
+const disableAutoInit = Boolean(esFormsLocalization.formDisableAutoInit) ?? false;
+
 // Load window form no matter what the option is set.
 window['esForms'] = {
 	events: FORM_EVENTS,
 	selectors: FORM_SELECTORS,
 	dataAttributes: FORM_DATA_ATTRIBUTES,
 	formSelector: selector,
+	initAll: () => {
+		initAll(true);
```

Comment:
Sorry to send it now. Might be good to fall back on default but to have an option to pass the option

```
	initAll: (manual = true) => {
		initAll(manual);

Reply:
if you are triggering this method this will we manual in any way so there is no need for this here